lib/resque_solo/queue.rb

```diff
@@ -60,8 +60,13 @@ def destroy(queue, klass, *args)
       end
 
       def cleanup(queue)
-        keys = redis.keys("solo:queue:#{queue}:job:*")
-        redis.del(*keys) if keys.any?
+        "0".tap do |cursor|
+          loop do
+            cursor, keys = redis.scan(cursor, match: "solo:queue:#{queue}:job:*")
+            redis.del(keys) if keys.count > 0
```

Comment:
I think this may need to change back to `redis.del(*keys) if keys.any?` (notice the `*` in `del(*keys)`). Might as well throw in the `keys.any?` change you mentioned in slack while you're at it.

Reply:
yah I think your right about *keys